### ISSUE TITLE
How to display a labels column?

### ISSUE THREAD

**Issue by claremacrae** [nathonius/obsidian-github-link#119]
### Purpose

I use a lot of labels on my repos - as well as  being useful for searching, it helps with glanceability as well - I can first look at the categories in the labels, and then the description...

### Experimentation

From the source code I thought there might be a `labels` column, so I tried it out:

~~~
```github-query
outputType: table
queryType: issue
query: "repo:obsidian-tasks-group/obsidian-tasks is:open is:issue label:\"status: in progress\""
columns: [number, title, author, created, status, labels]
```
~~~

It did add a Labels column, but it shows all the data for the labels so isn't readable enough to use...

![image](https://github.com/nathonius/obsidian-github-link/assets/4840096/837e8bd2-8219-4df1-8fea-af2890f2d39b)

### Ideas

Seeing that info:

- at a minimum it would be great for a comma-separated list of the issue names, such as:
    - `"priority: high", "scope: postpone", "scope: user interface", "status: in progress", "type: enhancement"`
- a nice-to-have would be for the label colours to be applied to that text...

**Comment by nathonius** [comment]
Oops, looks like I missed handling that data type. These should definitely display in then same way they do on github. Will get it added in the next release.

**Comment by claremacrae** [comment]
Super thanks...
`Assignee` does the same - would you like another feature request for that?
![image](https://github.com/nathonius/obsidian-github-link/assets/4840096/69cf9ab0-ac83-45f9-9e3c-8242a06a3214)

**Comment by claremacrae** [comment]
> in the meantime, you can display the text of a single label by adding `labels.0.name` as a column.

Super - thanks.

![image](https://github.com/nathonius/obsidian-github-link/assets/4840096/14eb4e49-965e-409c-ac85-ca3e37a361cf)

**Comment by nathonius** [comment]
> Super thanks...
> `Assignee` does the same - would you like another feature request for that?
> ![image](https://github.com/nathonius/obsidian-github-link/assets/4840096/69cf9ab0-ac83-45f9-9e3c-8242a06a3214)
> 

Yes please. 😁

**Comment by nathonius** [comment]
@claremacrae released in v1.0.0. Thanks for your patience. 😄
